fbw-a380x/src/systems/instruments/src/SD/Pages/Press/elements/OutflowValve.tsx

```diff
@@ -1,17 +1,30 @@
 import React, { memo } from 'react';
 import { useSimVar } from '@instruments/common/simVars';
-import { EngineNumber, Position } from '@instruments/common/types';
+import { EngineNumber, Position, ValidRedundantSystem } from '@instruments/common/types';
 import { GaugeComponent, GaugeMarkerComponent } from '@instruments/common/gauges';
+import { useArinc429Var } from '@flybywiresim/fbw-sdk';
 
-const OutflowValve: React.FC<Position & EngineNumber> = memo(({ x, y, engine }) => {
+const OutflowValve: React.FC<Position & EngineNumber & ValidRedundantSystem> = memo(({ x, y, engine, system }) => {
     const ofradius = 52;
 
     const [flightPhase] = useSimVar('L:A32NX_FWC_FLIGHT_PHASE', 'enum', 1000);
-    const [outflowValueOpenPercentage] = useSimVar(`L:A32NX_PRESS_OUTFLOW_VALVE_${engine}_OPEN_PERCENTAGE`, 'percent', 500);
+
+    const [manOutflowValueOpenPercentage] = useSimVar(`L:A32NX_PRESS_MAN_OUTFLOW_VALVE_${engine}_OPEN_PERCENTAGE`, 'percent', 500);
+
+    const outflowValueOpenPercentage = useArinc429Var(
+        `L:A32NX_PRESS_OUTFLOW_VALVE_${engine}_OPEN_PERCENTAGE_B${system}`,
+        500,
+    ).valueOr(manOutflowValueOpenPercentage);
+
+    const [dualChannelFailure] = useSimVar(`L:A32NX_PRESS_OCSM_${engine}_CHANNEL_1_FAILURE`, 'bool', 500)
```

Comment:
**issue:** Hooks should not be conditionally called. See the [rules of hooks](https://react.dev/reference/rules/rules-of-hooks)

Reply:
I've changed all instances where I was calling `useArinc429Var` with a `.valueOr`, as per the Press page in the A32NX code.